app/controllers/deliveries_controller.rb

```diff
@@ -1,8 +1,6 @@
 # encoding: utf-8
 class DeliveriesController < ApplicationController
 
-  before_filter :find_supplier, :exclude => :fill_new_stock_article_form
```

Comment:
Oops, that is an unrelated cleanup. I just found it by grep'ing "stock_article_form". Should I rebase and remove that change?

Reply:
OK, I will rebase and omit this change because it is not related to the rest.

Let us finish some discussions here for the moment.